### PR TITLE
Update Content.js

### PR DESCRIPTION
**Pull request by stefantsonev2019**
A PR description here.

### CODE REVIEW

File: Content.js [hunk 1]
```diff
@@ -66,8 +66,7 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
           <button className='brand cta' onClick={ () => comment('addSingleComment') }>
             Add single comment
           </button> }
-        { (!isEditing && handler.startReview) &&
-          <button className='brand cta' onClick={ () => comment('startReview') }>Start review</button> }
+          <button className='brand cta' onClick={ () => comment('startReview') }>Start review</button>
       </div> }
```
Comment (krasimir) [inline Content.js]: I like it.

File: Content.js [hunk 2]
```diff
@@ -66,8 +66,7 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
           <button className='brand cta' onClick={ () => comment('addSingleComment') }>
             Add single comment
           </button> }
-        { (!isEditing && handler.startReview) &&
-          <button className='brand cta' onClick={ () => comment('startReview') }>Start review</button> }
+          <button className='brand cta' onClick={ () => comment('startReview') }>Start review</button>
```
Comment (krasimir) [inline Content.js]: One more.

File: Content.js [hunk 3]
```diff
@@ -66,8 +66,7 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
           <button className='brand cta' onClick={ () => comment('addSingleComment') }>
             Add single comment
           </button> }
-        { (!isEditing && handler.startReview) &&
-          <button className='brand cta' onClick={ () => comment('startReview') }>Start review</button> }
```
Comment (krasimir) [inline Content.js]: You sure we have to delete it?

File: Root.js [hunk 4]
```diff
@@ -5,9 +5,9 @@ import Header from './Header';
 function App() {
   return (
     <div>
-      <Header>Hello world</Header>
+      <Header>Hey, there!</Header>
     </div>
```
Comment (krasimir) [inline Root.js]: Good one.

**Review by krasimir — APPROVED** [review]

File: Content.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /* A
-comment here
+Comment here.
 */
```
Comment (krasimir) [inline Content.js]: C1

File: Content.js (hunk 2 above)
Comment (krasimir) [inline Content.js]: C2

File: Content.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /* A
-comment here
+Comment here.
```
Comment (krasimir) [inline Content.js]: c1

File: Content.js (hunk 2 above)
Comment (krasimir) [inline Content.js]: c2

File: Content.js (hunk 5 above)
Comment (krasimir) [inline Content.js]: test

File: Content.js (hunk 6 above)
Comment (krasimir) [inline Content.js]: aaa

File: Content.js (hunk 3 above)
Comment (krasimir) [inline Content.js]: ddd

File: Content.js (hunk 6 above)
Comment (krasimir) [inline Content.js]: yu

File: Content.js (hunk 2 above)
Comment (krasimir) [inline Content.js]: uuu

File: Content.js (hunk 6 above)
Comment (krasimir) [inline Content.js]: C1

File: Root.js [hunk 7]
```diff
@@ -5,9 +5,9 @@ import Header from './Header';
 function App() {
   return (
     <div>
-      <Header>Hello world</Header>
+      <Header>Hey, there!</Header>
```
Comment (krasimir) [inline Root.js]: C2

**Review by krasimir — CHANGES_REQUESTED** [review]
I think that it is cool.

File: Root.js [hunk 8]
```diff
@@ -5,9 +5,9 @@ import Header from './Header';
 function App() {
```
Comment (krasimir) [inline Root.js]: Nice

Comment (krasimir) [reply Root.js]: Second.

File: Content.js [hunk 9]
```diff
@@ -1,5 +1,5 @@
 /* A
-comment here
```
Comment (krasimir) [inline Content.js]: TUTUT

**Review by krasimir — APPROVED** [review]
I'm happy

File: Root.js (hunk 7 above)
Comment (krasimir) [reply Root.js]: OOO

Comment (krasimir) [reply Root.js]: 444

**Review by krasimir — APPROVED** [review]

File: Root.js [hunk 10]
```diff
@@ -5,9 +5,9 @@ import Header from './Header';
...
   return (
     <div>
-      <Header>Hello world</Header>
```
Comment (krasimir) [inline Root.js]: Why?

File: Root.js (hunk 7 above)
Comment (krasimir) [reply Root.js]: That's indeed a good suggestion!

**Review by krasimir — CHANGES_REQUESTED** [review]
more!

**Review by krasimir — COMMENTED** [review]
A new one.

**Comment by krasimir** [comment]
Just a comment.

File: Root.js (hunk 7 above)
Comment (stefantsonev2019) [inline Root.js:8]: One more from another user.

**Comment by krasimir** [comment]
ONLYYYY

File: Root.js (hunk 10 above)
Comment (krasimir) [reply Root.js]: hhhh

Comment (krasimir) [reply Root.js]: tyr

**Review by krasimir — COMMENTED** [review]
blah

**Review by krasimir — APPROVED** [review]

**Comment by krasimir** [comment]
null

**Review by krasimir — APPROVED** [review]

File: Content.js [hunk 11]
```diff
@@ -1,5 +1,5 @@
 /* A
```
Comment (stefantsonev2019) [inline Content.js:1]: What you think!!!!!!!?

**Comment by krasimir** [comment]
909010

**Comment by krasimir** [comment]
1001

**Comment by krasimir** [comment]
1002

**Comment by krasimir** [comment]
Looks good, I'm thinking that this could be merged.

**Comment by krasimir** [comment]
I would vote for this notation. So instead of `setContact` I would prefer `withContact`. I like the sequence of `withSomething().withSomething().build()`. It kinda speaks "create my thing with this and that". While `setSomething().setSomething().build()` tells "I have something created and I'm setting this and that and at the end ... oh wait I'm building it now".

File: Content.js (hunk 11 above)
Comment (krasimir) [reply Content.js:1]: I think it is nice.

**Comment by krasimir** [comment]
200

**Comment by krasimir** [comment]
Go go

**Comment by krasimir** [comment]
KKOL

File: Content.js (hunk 3 above)
Comment (stefantsonev2019) [inline Content.js:70]: Good one.

Comment (krasimir) [reply Content.js:70]: Y

Comment (stefantsonev2019) [reply Content.js:70]: Y3

**Comment by krasimir** [comment]
That's a good feedback imo #2. This is a test with commit 8a2651c and this one e5ab6d29ab382d537d8ceb51cbb4926a9b73c474

**Pull request merged**